### ISSUE TITLE
xtb 6.6.0 gets stuck with hessian calculation

### ISSUE THREAD

**Issue by ToKa96** [grimme-lab/xtb#777]
**Describe the bug**
xtb 6.6.0 get stuck with hessian calculation last printed lines are:
~~~
           ------------------------------------------------- 
          |                Numerical Hessian                |
           ------------------------------------------------- 
step length          :   0.00500
SCC accuracy         :   0.30000
Hessian scale factor :   1.00000
frozen atoms in %    :   0.00000    0
RMS gradient         :   0.00026
~~~
after those it gets stuck. Same input same machine but using xtb v6.5.1 works fine.

**To Reproduce**

1.  `conda create -n xtb_6.6.0 -c conda-forge xtb=6.6.0`
2.  `conda activate xtb_6.6.0`
3.1 use input file `h2o.xyz`:

~~~
3

O 0 0 0
H 0 0 1
H 0 1 0
~~~
3.2 (optional did not make a difference in my tests):
~~~
ulimit -s unlimited
export OMP_STACKSIZE=4G
export OMP_NUM_THREADS=<ncores>,1
export OMP_MAX_ACTIVE_LEVELS=1
export MKL_NUM_THREADS=<ncores>
~~~
I used `<ncores>=12` or `<ncores>=16` depending on the machine

4.a) `xtb h2o.xyz --hess` -> gets stuck
4.b) 
  1.  `xtb h2o.xyz --opt` -> works fine
  2. `xtb xtbopt.xyz --hess` -> gets stuck
 
Output of the above with `--verbose` see xtb_6.6.0_fail.log

**Expected behaviour**
Some form of termination rather than getting stuck.

**Additional context**
same procedure as above on the same machines works fine when using xtb v6.5.1 installed via `conda create -n xtb_6.5.1 -c conda-forge xtb=6.5.1` 

Do you need any additional information?
Thank you for your time and help! 
[xtb_6.6.0_fail.log](https://github.com/grimme-lab/xtb/files/10911777/xtb_6.6.0_fail.log)

**Comment by MtoLStoN** [comment]
This is a known bug that was fixed in #774. We should probably release a bug fix release soon. You can either build yourself from the master branch or use the pre-built binaries.

I will leave this open for now until we release xTB 6.6.1.

**Comment by MtoLStoN** [comment]
With the Release of xTB 6.6.1, this should be fixed now in the latest stable Release.
